modules/artifact/src/main/java/org/mule/runtime/module/artifact/classloader/ArtifactClassLoaderFilter.java

```diff
@@ -70,9 +87,9 @@ public boolean exportsClass(String className)
     public boolean exportsResource(String name)
     {
         checkArgument(name != null, "Resource name cannot be null");
-        final String resourcePackage = getResourceFolder(name);
+        final String sanitizeResourceName = sanitizeResourceName(name);
 
-        return exportedResourcePackages.contains(resourcePackage);
+        return exportedResources.contains(sanitizeResourceName);
```

Comment:
container supports exporting folders and files, changing what is exported in every module is not part of this PR, but it will be part of future changes once the API is clearly defined